schema/schema.graphql

```diff
@@ -11,7 +11,14 @@ const utils = require("./utils.graphql");
 
 const typeDefs = gql`
   directive @auth on FIELD_DEFINITION
+  directive @role(requires: Role) on FIELD_DEFINITION
 
+  enum Role {
```

Comment:
The user.graphql file already contains the `UserType` enum. Please remove this duplicate code and use that

Reply:
Fixed